build/build.sh

```diff
@@ -115,51 +136,63 @@ if [[ $_RUN_STEP_NODE = true ]]; then
     fi
     [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
     nvm install "$NODE_VERSION"
-    end_time=$(date +%s%N)
-    echo "=== [⬆️] Setup Node.js: $(((end_time - start_time) / 1000000))ms ==="
+    # re-enable debugging output
+    enable_debugging
```

Comment:
Why enable printing and disable it without any output?

Reply:
the enabling and disabling at the beginning and end of each step is added automatically. As I don't want to spam the console too much, I've manually disabled the logging.

I will improve the build script to remove these unnecessary enabling/disabling commands.

Reply:
this is resolved. I improved the script generation :)